src/en/installconfig-gui.md

```diff
@@ -0,0 +1,118 @@
+Title: Access the GUI
+TODO: Update images
+	
+# Access the GUI
+
+Once MAAS is installed, you'll need to create an administrator account:
+
+```bash
+sudo maas createadmin --username=root --email=MYEMAIL@EXAMPLE.COM
```

Comment:
So I believe we should be pushing the GUI by default. All CLI alternatives should be given later. Example:

"To create the superuser vi the CLI see 'Using the CLI#create-superuser' ".

The same goes for all other CLI mentions on this page. We can let it go for now if you want and leave a TODO.

Ditto for page installconfig-images.md